src/common/StatusBadge/UserStatusBadge.tsx

```diff
@@ -24,7 +25,7 @@ export const UserStatusBadge = ({ status }: { status: UserStatus }) => {
     <Badge
       className={`capitalize flex justify-center ${background} ${text} hover:${background} hover:${text} hover:cursor-pointer`}
     >
-      {status}
+      {capitalization(status)}
```

Comment:
@muhammad-bilal-ali-khokhar  Why do we need to use this `capitalization` function when it's already handled with CSS?